sandbox.html

```diff
@@ -136,6 +136,7 @@
       showPage: data => cadenzaClient.show({ page: data.page }, getOptions(data)),
       expandNavigator: data => cadenzaClient.expandNavigator(data.expandNavigator === 'on'),
       showMap: data => cadenzaClient.showMap(data.embeddingTargetId, getOptions(data)),
+      setLayerVisibility: data => cadenzaClient.setLayerVisibility(data.layer.split(','), data.visibility === 'on'),
```

Comment:
In the other place where we ask for a layer, we parse JSON. I think this should be done consistently and it should be clear for the user what to enter.

Reply:
In that other place we ask for multiple layers. So, JSON here?

Reply:
At least you should add a hint below the input (<small>), what the expectation is.

Reply:
I tried making it JSON. But I can't seem to find the right json syntax for a single array (not a json object).

Reply:
```
["groupName", "layerName"]
```
?

Reply:
.... I was trying with single quotes, and that's not valid json of course :smile:  :brain: :gun: